Source/BrackeysGameJam2020/Private/EnemyAIController.cpp

```diff
@@ -0,0 +1,41 @@
+
+
+#include "EnemyAIController.h"
+#include "EnemyBase.h"
+#include "BehaviorTree/BehaviorTree.h"
+#include "BehaviorTree/Blackboard/BlackboardKeyType_Enum.h"
+#include "BehaviorTree/BehaviorTreeComponent.h"
+#include "BehaviorTree/BlackboardComponent.h"
+
+AEnemyAIController::AEnemyAIController()
+{
+	BehaviorComp = CreateDefaultSubobject<UBehaviorTreeComponent>(TEXT("BehaviorComp"));
+	BlackboardComp = CreateDefaultSubobject<UBlackboardComponent>(TEXT("BlackboardComp"));
+
+	TargetKeyName = "Target";
+}
+
+void AEnemyAIController::OnPossess(APawn * pawn)
```

Comment:
Just a small change to unify the code to Pascal Case.
```suggestion
void AEnemyAIController::OnPossess(APawn * Pawn)
```